skimage/_shared/_deprecators.py

```diff
@@ -0,0 +1,11 @@
+from functools import partial
+from ..version import __version__
+from wabisabi import kwonly_change, default_parameter_change
+
+kwonly_change = partial(kwonly_change,
+                        library_name='scikit-image',
+                        current_library_version=__version__)
```

Comment:
shouldn't the version number hard coded for each deprecation?

Reply:
This corresponds 0.17, not 1.0 today

Reply:
I know, but I though that it will be used for docstring mangling... I mean to document when the changes have been introduced...